db/migrate/20161226195633_create_tasks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks, id: :uuid do |t|
```

Comment:
Nice! I like that you saw we were using `uuid` on projects and used it here as well.